templates/user/notification/notification_div.tmpl

```diff
@@ -59,24 +59,24 @@
 									{{end}}
 								</td>
 								<td class="eleven wide">
-									<a class="item" href="{{.Link}}">
+									<a class="item issue-title muted" href="{{.Link}}">
```

Comment:
Hmm ... I don't understand why these links should be `muted`.

Indeed they are really links.

So, the question is: what kinds of links should be muted, while what kinds shouldn't? Could there be a guideline?

Reply:
It's part of a overall removal of link colors that we have been doing across the UI in recent changes. More of a stylistical thing.

Reply:
Sorry I didn't get the point.

Do you mean that: every link should be muted?

Reply:
Does it mean that we should set muted style as default, and introduce `unmuted` for rare cases?

Reply:
Could do, yes, but it will be a massive refactor. Also probably find a better name.